app/templates/application.hbs

```diff
@@ -1,3 +1,5 @@
+{{title 'OSF | Preprints' separator=' | '}}
```

Comment:
IS the | in between OSF and Preprints necessary? I realize that this is the structure for regular osf pages like ```OSF | Home``` but I think in this case, since OSF Preprints is a service and its title already has OSF, we can remove this.